lib/logstash/filters/fingerprint.rb

```diff
@@ -134,6 +134,9 @@ class << self; alias_method :fingerprint, :fingerprint_murmur3_128; end
     when :PUNCTUATION
       # nothing
     else
+      # force the resolution of OpenSSL class to avoid errors when loaded in multithreaded
+      # #select_digest method. https://github.com/logstash-plugins/logstash-filter-fingerprint/issues/75
+      OpenSSL::Digest::MD5
```

Comment:
When `@method` is something other than `:MD5`, `select_digest` will resolve to a different class. 

We may need to refactor `Fingerprint#select_digest` to return the _class_ so we can use it here to ensure that the digest class we need is ready, and then send `Class#new` to it in `Fingerprint#fingerprint_openssl`.

Reply:
Thanks for suggesting it! 👍  Fixed with commit 9454adaab442240164a879efb78512119489b3e7